source/slang/slang-emit-c-like.cpp

```diff
@@ -1161,6 +1162,40 @@ bool CLikeSourceEmitter::shouldFoldInstIntoUseSites(IRInst* inst)
     return true;
 }
 
+void CLikeSourceEmitter::emitDereferenceOperand(IRInst* inst, EmitOpInfo const& outerPrec)
+{
```

Comment:
The obvious alternative here would be to have `emitDereferenceOperand` be a customization point on `CLikeSourceEmitter` where subtypes override these operations. I think you are probably making the right choice here since there are currently only two options (actually emit the op, or ignore it).

Reply:
This was originally a virtual function that can be customized like you said. But then I find out it really doesn't give you much and turn it back into just an ordinary function.